crates/astria-sequencer/src/mempool/transactions_container.rs

```diff
@@ -278,17 +433,30 @@ pub(super) trait TransactionsForAccount: Default {
         current_account_nonce: u32,
     ) -> bool;
 
+    /// Returns `Ok` if adding `ttx` would not break the balance precondition, i.e. enough
+    /// balance to cover all transactions in `CostsCovered` mode.
```

Comment:
I was trying to do the naming similar to the `SequentialNonces` mode we have but for saying the container only contains transactions that are covered by the account's balances. On reflection `CostsCovered` isn't the same as the `SequentialNonces` case because we have to reclean to stay in `CostsCovered` when `SequentialNonces` is always true.

Happy to remove the comments if you think they are misleading.

Reply:
I guess if you're ok to remove just the " in `CostsCovered` mode" part, that would make more sense to me.